designer/client/conditions/inline-conditions-definition-values.js

```diff
@@ -49,12 +55,31 @@ function SelectValues (props) {
   )
 }
 
+function customValueComponent (fieldType, operator) {
+  const operatorConfig = getOperatorConfig(fieldType, operator)
+  const absoluteDateTimeRenderFunctions = {
+    'DateField': (value, updateValue) => <AbsoluteDateValues value={value} updateValue={updateValue} />,
+    'DatePartsField': (value, updateValue) => <AbsoluteDateValues value={value} updateValue={updateValue} />,
+    'DateTimeField': (value, updateValue) => <AbsoluteDateTimeValues value={value} updateValue={updateValue} />,
+    'DateTimePartsField': (value, updateValue) => <AbsoluteDateTimeValues value={value} updateValue={updateValue} />,
+    'TimeField': (value, updateValue) => <AbsoluteTimeValues value={value} updateValue={updateValue} />
+  }
+  const dateTimeFieldTypes = Object.keys(absoluteDateTimeRenderFunctions).includes(fieldType)
```

Comment:
`absoluteDateTimeRenderFunctions[fieldType]`? 

`includes` will only return a bool, the const name makes it sound like an array of fieldTypes

Reply:
Fixed